data/json/items/ammo/762x25.json

```diff
@@ -0,0 +1,58 @@
+ [
+  {
+    "id": "762x25",
+    "type": "AMMO",
+    "name": "7.62x25mm JHP",
+    "name_plural": "7.62x25mm JHP",
+    "description": "A commercial version of the 7.62x25mm cartridge created for the armed forces of Soviet Russia.  It was derived from the 7.63x25mm cartridge used by the C96 pistol.",
+    "weight": 9,
+    "volume": 1,
+    "price": 3000,
+    "count": 40,
+    "stack_size": 40,
+    "material": ["steel", "powder"],
+    "symbol": "=",
+    "color": "light_gray",
+    "ammo_type": "762x25",
+    "casing": "762_casing",
+    "damage": 24,
+    "pierce": 2,
+    "range": 13,
+    "dispersion": 120,
+    "recoil": 195,
+    "effects": [ "COOKOFF" ]
+  },
+  {
+    "id" : "762_25hot",
+    "copy-from": "762_25",
+    "type" : "AMMO",
+    "name" : "7.62x25mm FMJ hot load",
+    "name_plural" : "7.62x25mm FMJ hot load",
+    "description" : "A high-pressure commercial version of the 7.62x25mm cartridge, loaded with a 85 gr. FMJ bullet.  It is more powerful than the original.",
+    "price": 3500,
+    "relative": {
+        "damage": 3,
+        "range": 2,
+        "pierce": 3,
+        "recoil": 45
+    }
+  },
+  {
+    "id" : "762_25typeP",
+    "copy-from": "762_25",
+    "type" : "AMMO",
+    "name" : "7.62x25mm Type P",
+    "name_plural": "7.62x25mm Type P",
+    "description": "A subsonic cartridge derived from the 7.62x25mm, designed for silenced firearms.  It offers good armor penetration at the cost of slightly less damage.",
+    "weight": 12,
+    "price": 5000, "//": "Stopgap price of $1 per.  Anti-China prejudice in the pre-Cataclysm US may have made this ammo tougher to come by.",
+    "count": 50,
+    "loudness": 60,
```

Comment:
If you reduce `damage` by this much you probably don't need `loudness` here at all